src/docs/sphinx/developerGuide/Contributing/InstallWin.rst

```diff
@@ -0,0 +1,266 @@
+
+Note that in addition to the detached flag (*-d*) and the name tage (*--name*), we provide *Docker* with the port the conatiner should associated to
+communicate with ssh port 22, as well as a binding between a host mount point (*D:/install_geosx_docker/*) and a container mount point (*/app*) to have a peristent storage
+for our development/geosx builds. More details on the `--mount options <https://docs.docker.com/storage/bind-mounts/>`_
+
+There exact same steps can be achieve using th *Docker Desktop* GUI in the image tabs, clicking on the run button and filling the same information in the interface,
+
+.. image:: win_install/win_docker_container.png
+   :width: 45%
+
+Coming back to our *PowerShell* terminal, we can check that our container is running and try to ssh to it.
+
+.. code:: shell
+
+    PS > docker ps -a
+    CONTAINER ID   IMAGE                                 COMMAND               CREATED                  STATUS          PORTS                                     NAMES
+    1efffac66c4c   remote-dev-ubuntu18.04-gcc8:156-642   "/usr/sbin/sshd -D"   Less than a second ago   Up 18 seconds   0.0.0.0:64000->22/tcp, :::64000->22/tcp   remote-dev-ubuntu18.04-gcc8-156-642
+    PS > ssh root@localhost -p 64000
+    Enter passphrase for key 'C:\************.ssh/id_rsa':
+    Welcome to Ubuntu 18.04.5 LTS (GNU/Linux 5.10.16.3-microsoft-standard-WSL2 x86_64)
+
+     * Documentation:  https://help.ubuntu.com
+     * Management:     https://landscape.canonical.com
+     * Support:        https://ubuntu.com/advantage
+    This system has been minimized by removing packages and content that are
+    not required on a system that users do not log into.
+
+    To restore this content, you can run the 'unminimize' command.
+
+    The programs included with the Ubuntu system are free software;
+    the exact distribution terms for each program are described in the
+    individual files in /usr/share/doc/*/copyright.
+
+    Ubuntu comes with ABSOLUTELY NO WARRANTY, to the extent permitted by
+    applicable law.
+
+    root@b105f9ead860:~# cd /app && ls
+
+We are now logged into our container and can start :ref:`QuickStart`.
+
+.. note::
+    You might be prompted that you miss certificates to clone, this can be reolved installing *ca-certificates* and updating them
+
+        PS > apt install ca-certificates && update-ca-certificates
+
+.. note::
+    It might occur that *git-lfs* is missing then install it,
+
+        PS > apt install git-lfs
+
+From there you should be able to develop in your container or access it from an IDE, e.g. `VSCode <https://code.visualstudio.com/docs/remote/ssh>`_
+or `MSVC19 <https://docs.microsoft.com/en-us/cpp/linux/connect-to-your-remote-linux-computer?view=msvc-160>`_.
+
+5. Running a case
+-------------------
+
+    Once the code is configured and compiled, let us check status of the build,
+
+.. code:: shell
+
+    root@b105f9ead860:~# cd [path-to-build]/ && ./bin/geosx --help
```

Comment:
How do you connect to this container?

Reply:
I think it is in the block code above (section 4, l.193)  but I can repeat the line here.

Reply:
Sorry I missed it 🙈 . Just leave it like this then!